src/third-party/PhishingController.ts

```diff
@@ -75,6 +76,7 @@ export class PhishingController extends BaseController<PhishingConfig, PhishingS
     this.defaultState = {
       phishing: DEFAULT_PHISHING_RESPONSE,
       whitelist: [],
+      tolerance: 2,
     };
     this.detector = new PhishingDetector(this.defaultState.phishing);
```

Comment:
The `tolerance` value needs to be passed into the constructor here, via this `phishing` property. Maybe we could add this to `DEFAULT_PHISHING_RESPONSE`? I don't think this is working as currently written.

Reply:
prolly should just be a default over on eth-phishing-detect?